gameoflife/src/main/kotlin/com/anaplan/engineering/kazuki/gameoflife/Conway.kt

```diff
@@ -0,0 +1,142 @@
+package com.anaplan.engineering.kazuki.gameoflife
+
+import com.anaplan.engineering.kazuki.core.*
+import com.anaplan.engineering.kazuki.gameoflife.Conway_Module.mk_Point
+import com.anaplan.engineering.kazuki.gameoflife.Conway_Module.mk_Population
+
+@Module
+object Conway {
+
+    private const val Generate = 3 // Number of neighbours to cause generation
+    private val Survive = mk_Set(2, 3) // Numbers of neighbours to ensure survival, else death
+    private const val maxNeigh = 8
```

Comment:
⛏️ The name of every other constant appears to be fully spelled out, so this would be more consistent.
```suggestion
    private const val maxNeighbours = 8
```